jobs/build/ocp4/Jenkinsfile

```diff
@@ -119,6 +119,13 @@ node {
         }
         stage("report success") { build.stageReportSuccess() }
     } catch (err) {
+
+        if ( buildPlan.is_permitted == false ) {
+            echo 'Exiting because this build is not permitted: ${err}'
```

Comment:
Could it go in the `currentBuild.description` too?

Reply:
This is now done in `assertBuildPermitted`